src/index.ts

```diff
@@ -93,7 +93,6 @@ const program = new Command(packageJson.name)
     .option('-w, --websiteName <websiteName>', 'Name of the website, default is solution name')
     .option('-u, --username <username>', 'User name for the virtual machine that will be created as part of the solution')
     .option('-p, --password <password>', 'Password for the virtual machine that will be created as part of the solution')
-    .option('--sshFilePath <sshFilePath>', 'Path to the ssh file path that will be used by standard deployment')
```

Comment:
remove same option in the launch settings under .vscode folder

Reply:
Done